# HG changeset patch
# User Neil Hodgson <nyamatongwe@gmail.com>
# Date 1511240678 -39600
# Node ID 8760026bba6ab6a5c6ee3d2d7bd7a291ecb74574
# Parent  85205da6ec1b8eeb14c3edc94064fb0d8f685377
Update marshalling of signals from Scintilla, changing deprecated NONE to VOID.
Has no effect on clients as NONE was an alias for VOID.

diff -r 85205da6ec1b -r 8760026bba6a gtk/scintilla-marshal.c
--- a/gtk/scintilla-marshal.c	Mon Nov 20 12:24:51 2017 +1100
+++ b/gtk/scintilla-marshal.c	Tue Nov 21 16:04:38 2017 +1100
@@ -1,6 +1,4 @@
-
-#include	<glib-object.h>
-
+#include <glib-object.h>
 
 #ifdef G_ENABLE_DEBUG
 #define g_marshal_value_peek_boolean(v)  g_value_get_boolean (v)
@@ -48,8 +46,7 @@
 #define g_marshal_value_peek_variant(v)  (v)->data[0].v_pointer
 #endif /* !G_ENABLE_DEBUG */
 
-
-/* NONE:INT,OBJECT (scintilla-marshal.list:1) */
+/* VOID:INT,OBJECT (scintilla-marshal.list:1) */
 void
 scintilla_marshal_VOID__INT_OBJECT (GClosure     *closure,
                                     GValue       *return_value G_GNUC_UNUSED,
@@ -58,13 +55,13 @@
                                     gpointer      invocation_hint G_GNUC_UNUSED,
                                     gpointer      marshal_data)
 {
-  typedef void (*GMarshalFunc_VOID__INT_OBJECT) (gpointer     data1,
-                                                 gint         arg_1,
-                                                 gpointer     arg_2,
-                                                 gpointer     data2);
+  typedef void (*GMarshalFunc_VOID__INT_OBJECT) (gpointer data1,
+                                                 gint arg1,
+                                                 gpointer arg2,
+                                                 gpointer data2);
+  GCClosure *cc = (GCClosure *) closure;
+  gpointer data1, data2;
   GMarshalFunc_VOID__INT_OBJECT callback;
-  GCClosure *cc = (GCClosure*) closure;
-  gpointer data1, data2;
 
   g_return_if_fail (n_param_values == 3);
 
@@ -86,7 +83,7 @@
             data2);
 }
 
-/* NONE:INT,BOXED (scintilla-marshal.list:2) */
+/* VOID:INT,BOXED (scintilla-marshal.list:2) */
 void
 scintilla_marshal_VOID__INT_BOXED (GClosure     *closure,
                                    GValue       *return_value G_GNUC_UNUSED,
@@ -95,13 +92,13 @@
                                    gpointer      invocation_hint G_GNUC_UNUSED,
                                    gpointer      marshal_data)
 {
-  typedef void (*GMarshalFunc_VOID__INT_BOXED) (gpointer     data1,
-                                                gint         arg_1,
-                                                gpointer     arg_2,
-                                                gpointer     data2);
+  typedef void (*GMarshalFunc_VOID__INT_BOXED) (gpointer data1,
+                                                gint arg1,
+                                                gpointer arg2,
+                                                gpointer data2);
+  GCClosure *cc = (GCClosure *) closure;
+  gpointer data1, data2;
   GMarshalFunc_VOID__INT_BOXED callback;
-  GCClosure *cc = (GCClosure*) closure;
-  gpointer data1, data2;
 
   g_return_if_fail (n_param_values == 3);
 
diff -r 85205da6ec1b -r 8760026bba6a gtk/scintilla-marshal.h
--- a/gtk/scintilla-marshal.h	Mon Nov 20 12:24:51 2017 +1100
+++ b/gtk/scintilla-marshal.h	Tue Nov 21 16:04:38 2017 +1100
@@ -1,30 +1,30 @@
+/* This file is generated, all changes will be lost */
+#ifndef __SCINTILLA_MARSHAL_MARSHAL_H__
+#define __SCINTILLA_MARSHAL_MARSHAL_H__
 
-#ifndef __scintilla_marshal_MARSHAL_H__
-#define __scintilla_marshal_MARSHAL_H__
-
-#include	<glib-object.h>
+#include <glib-object.h>
 
 G_BEGIN_DECLS
 
-/* NONE:INT,OBJECT (scintilla-marshal.list:1) */
-extern void scintilla_marshal_VOID__INT_OBJECT (GClosure     *closure,
-                                                GValue       *return_value,
-                                                guint         n_param_values,
-                                                const GValue *param_values,
-                                                gpointer      invocation_hint,
-                                                gpointer      marshal_data);
-#define scintilla_marshal_NONE__INT_OBJECT	scintilla_marshal_VOID__INT_OBJECT
+/* VOID:INT,OBJECT (scintilla-marshal.list:1) */
+extern
+void scintilla_marshal_VOID__INT_OBJECT (GClosure     *closure,
+                                         GValue       *return_value,
+                                         guint         n_param_values,
+                                         const GValue *param_values,
+                                         gpointer      invocation_hint,
+                                         gpointer      marshal_data);
 
-/* NONE:INT,BOXED (scintilla-marshal.list:2) */
-extern void scintilla_marshal_VOID__INT_BOXED (GClosure     *closure,
-                                               GValue       *return_value,
-                                               guint         n_param_values,
-                                               const GValue *param_values,
-                                               gpointer      invocation_hint,
-                                               gpointer      marshal_data);
-#define scintilla_marshal_NONE__INT_BOXED	scintilla_marshal_VOID__INT_BOXED
+/* VOID:INT,BOXED (scintilla-marshal.list:2) */
+extern
+void scintilla_marshal_VOID__INT_BOXED (GClosure     *closure,
+                                        GValue       *return_value,
+                                        guint         n_param_values,
+                                        const GValue *param_values,
+                                        gpointer      invocation_hint,
+                                        gpointer      marshal_data);
+
 
 G_END_DECLS
 
-#endif /* __scintilla_marshal_MARSHAL_H__ */
-
+#endif /* __SCINTILLA_MARSHAL_MARSHAL_H__ */
diff -r 85205da6ec1b -r 8760026bba6a gtk/scintilla-marshal.list
--- a/gtk/scintilla-marshal.list	Mon Nov 20 12:24:51 2017 +1100
+++ b/gtk/scintilla-marshal.list	Tue Nov 21 16:04:38 2017 +1100
@@ -1,2 +1,2 @@
-NONE:INT,OBJECT
-NONE:INT,BOXED
+VOID:INT,OBJECT
+VOID:INT,BOXED
